website/server/generate.js

```diff
@@ -19,29 +20,7 @@ server.noconvert = true;
 // Sadly, our setup fatals when doing multiple concurrent requests
 // I don't have the time to dig into why, it's easier to just serialize
 // requests.
-var queue = (function() {
```

Comment:
I was kind of proud of this function, I must have passed some interview question by writing it!